gcp/gke/variables.tf

```diff
@@ -1,21 +1,58 @@
-variable "name" {}
+variable "region" {
+  default = "us-central1"
+}
 
-variable "region" {}
+variable "environment" {
+  default = ""
+}
+
+variable "kubernetes_version" {
+  description = "Version of kubernetes to setup"
+  default     = "latest"
+}
+
+variable "costcenter" {}
 
 variable "project_id" {}
 
-variable "initial_node_count" { default = 1 }
+variable "cluster_name" {}
+
+variable "network" {}
+
+variable "subnetwork" {}
 
-variable "node_type" { default = "n1-standard-1" }
+variable "ip_range_pods" {
+  default = ""
+}
 
-variable "http_load_balancing_disabled" { default = false }
+variable "ip_range_services" {
+  default = ""
+}
 
-variable "istio_disabled" { default = true }
+variable "regional" {
+  description = "Configure cluster as a regional or zonal cluster"
+  default     = true
```

Comment:
do we really want regional by default? not opposed but we hadn't discussed this, and we're currently generally using zonal in gke, and regional costs more.

Reply:
Fixed in 7b1b3f5c949de5f14cb306d2e68b26350b63ff8e